build/ci/Dockerfile

```diff
@@ -5,7 +5,15 @@ WORKDIR /tmp
 # Install dependencies
 
 RUN apk update \
-    && apk add --update --no-cache bash curl git make gcc musl-dev docker
+    && apk add --update --no-cache bash curl git make gcc musl-dev docker tar
+
+# Configure python
```

Comment:
You can also add `pip install` command to speed up testing

Reply:
fixed